bin/faucet/src/handlers.rs

```diff
@@ -77,7 +77,7 @@ pub async fn get_tokens(
     // Ideally, we should export these note details with a more sensible `after_block_num`
     let bytes = NoteFile::NoteDetails {
         details: note_details,
-        after_block_num: 0,
+        after_block_num: client.get_chain_tip(),
```

Comment:
The comment above needs to be changed now that we execute against non-zero block.